README.md

```diff
@@ -0,0 +1,19 @@
+Hello-Microverse
+===============
```

Comment:
- [ ] ** Required:** Please make the appropriate changes requested by the previous TSE. Use the Microverse Template. Click the following link [REAMDE](https://github.com/microverseinc/readme-template) to have a better idea of how to build a descriptive README file.